lib/ocrypto/ec_key_pair.go

```diff
@@ -0,0 +1,103 @@
+package ocrypto
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+)
+
+type eccMode uint8
+
+const (
+	eccModeSecp256r1 eccMode = 0
+	eccModeSecp384r1 eccMode = 1
+	eccModeSecp521r1 eccMode = 2
+	eccModeSecp256k1 eccMode = 3
+)
+
+type EcKeyPair struct {
+	privateKey *ecdsa.PrivateKey
```

Comment:
Suggestion: Make it PrivateKey because KAS needs it. I did same thing for AsymEncryption

Reply:
Fixed